frontend-v2/src/components/Navbar/Navbar.tsx

```diff
@@ -0,0 +1,147 @@
+                <a
+                  href={doLogout ? "/logout" : "#"}
+                  onClick={(e) => { setDoLogout(true); if (!doLogout) { e.preventDefault() } }}
+                  onMouseLeave={() => setDoLogout(false)}
+                  className={`flex flex-row items-center justify-center rounded-xl gap-2 ${doLogout ? "hover:text-red-600 hover:bg-red-100" : "hover:bg-slate-200"}`}>
+                  <ArrowRightOnRectangleIcon className="w-6 h-12 py-3" />
+                </a>
+              </Tooltip>
+            </div>
+          </div>
+        </div>
+      ) : (
+        // Enlarged (non-collapsed) View
+        <div className="fixed flex flex-col w-72 h-screen gap-4 p-4 bg-gray-50 z-50">
+          {/* Logo */}
+          <div className="flex flex-row items-center justify-between h-10 p-2">
+            <a href="/">
+              <Image
+                src={logo}
+                width={33}
+                height={33}
+                alt="logo"
+                priority
+              />
+            </a>
+            <p className="text-xl font-semibold">Uni-lectives</p>
+            <BarsArrowDownIcon onClick={() => setCollapsed(true)} className="w-12 h-12 p-3 rotate-90 hover:bg-slate-200 rounded-xl" />
+          </div>
+          {/* Navbar Container */}
+          <div className="flex flex-col h-full w-full justify-between border-t-2 border-gray-200">
+            {/* Review Options */}
+            <div className="flex flex-col gap-3 items-left py-3">
+              <a className="flex flex-row items-center hover:bg-slate-200 rounded-xl" href="/">
+                <BookOpenIcon className="w-12 h-12 p-3" />
+                <span>Browse Courses</span>
+              </a>
+              <a className="flex flex-row items-center hover:bg-slate-200 rounded-xl" href="/user/zid">
+                <PencilSquareIcon className="w-12 h-12 p-3 hover:bg-slate-200 rounded-xl" />
+                <span>My Reviews</span>
+              </a>
+              <a className="flex flex-row items-center hover:bg-slate-200 rounded-xl" href="/terms-and-conditions">
+                <ShieldCheckIcon className="w-12 h-12 p-3 hover:bg-slate-200 rounded-xl" />
+                <span>Terms and Conditions</span>
+              </a>
+            </div>
+            {/* Account Options */}
+            <div className="flex flex-col gap-3 py-2 items-left">
+              <div className="flex flex-row justify-between gap-2">
+                <a className="flex flex-row w-full items-center hover:bg-slate-200 rounded-xl" href="user/zid">
+                  <UserCircleIcon className="w-12 h-12 p-3" />
+                  <span>Darian Lee</span>
+                </a>
+                <MoonIcon title="Dark Mode" className="w-12 h-12 p-3 hover:bg-slate-200 rounded-xl" />
+              </div>
+              <a
+                href={doLogout ? "/logout" : "#"}
+                onClick={(e) => { setDoLogout(true); if (!doLogout) { e.preventDefault() } }}
```

Comment:
extract logout logic to helper fn so `onClick` handler will become
```suggestion
                onClick={handleLogout}
```
then handleLogout would contain the logic to "log" the user out